### PR TITLE
Feature/webtrends

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of profiles-pr-104 in nhsuk-dev succeeded (http://profiles-pr-104.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of profiles-pr-104 in nhsuk-dev succeeded (http://profiles-pr-104.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of profiles-pr-104 in nhsuk-dev succeeded (http://profiles-pr-104.dev.c2s.nhschoices.net)

File: app/views/includes/analytics.nunjucks [hunk 1]
```diff
@@ -11,11 +11,9 @@
     ga('send', 'pageview');
   </script>
 
+  <script src="{{ SITE_ROOT }}/js/custom-ga.js" type="text/javascript"></script>
```
Comment (st3v3nhunt) [inline app/views/includes/analytics.nunjucks:14]: Move this script include to within the GA if statement as there isn't much point in having it load when GA isn't loaded

**Review by markysoft — APPROVED** [review]
LGTM

**Pull request merged**